code/controllers/subsystems/initialization/cuisine.dm

```diff
@@ -0,0 +1,30 @@
+
+/var/datum/controller/subsystem/cuisine/SScuisine
+
+/datum/controller/subsystem/cuisine
```

Comment:
Recipes should just be handled by being /decls rather than having the /datums loaded by this subsystem.

Reply:
UUUUUUUUUUUUUUUUUUUUURGH that can be done later. This is just moving it off of SSMachinery.

Reply:
I think it should be done in this PR, no one is going to do it later.

Reply:
Any update? I still don't believe using decls for this is feasible given how they're organised/accessed.